client/src/components/Search/Search.js

```diff
@@ -12,11 +12,6 @@ import * as moment from 'moment';
  */
 export default class Search extends React.Component {
 
-    handlePageChange(page) {
-        const { onPageChange } = this.props;
-        onPageChange(page);
-    }
-
```

Comment:
Why was this deleted? looks like it's still being called in the `Pagination` below